msdsl/function.py

```diff
@@ -77,6 +77,13 @@ def get_coeffs_cvxpy(self, func):
         addr_int = addr_real.astype(np.int)
         addr_frac = addr_real - addr_int
 
+        # add one final point at the end (otherwise the last coefficient may end up being zero,
```

Comment:
I found a bug, where the cvxpy implementation of functions didn't properly handle the very last segment of the lookup tables.  We didn't catch this before because the test function was zero at that point.